packages/@okta/vuepress-site/docs/reference/api/authn/index.md

```diff
@@ -91,9 +91,17 @@ The context object allows [trusted web applications](#trusted-application) such
 | ----------- | ----------------------------------------------------------------------------- | -------- | -------- | ------ | -------- | --------- | --------- |
 | deviceToken | A globally unique ID identifying the user's client device or user agent | String   | TRUE     | FALSE  | FALSE    |           | 32        |
 
-> **Note:** You must always pass the same `deviceToken` for a user's device with every authentication request for **per-device** or **per-session** Sign-On Policy Factor challenges. If the `deviceToken` is absent or does not match the previous `deviceToken`, the user is challenged every-time instead of **per-device** or **per-session**.
+> **Note:** You must always pass the same `deviceToken` for a user's device with every authentication request for **per-device** or **per-session** Sign-On Policy Factor challenges. If the `deviceToken` is absent or does not match the previous `deviceToken`, the user is challenged every-time instead of **per-device** or **per-session**. Similarly you must always pass the same `deviceToken` for user's device with every authentication request for **new device security behavior detection**. If the `deviceToken` is absent or does not match a recent `deviceToken` for the user, the request will be considered to be from a new device. For more information about this feature, see [Security Behavior Detection](https://help.okta.com/en/prod/Content/Topics/Security/proc-security-behavior-detection.htm).
 
-It is recommended that you generate a UUID or GUID for each client and persist the `deviceToken` as a persistent cookie or HTML5 localStorage item scoped to your web application's origin.
+##### Device Token Best Practices
```

Comment:
this section is mostly copied from the `##### Device Fingerprint Best Practices` section but keeps some of the language from the lines that were deleted here